### ISSUE TITLE
Disable sound through FoundryVTT

### ISSUE THREAD

**Issue by MythicalRollers** [frondeus/fvtt-syrin-control#104]
**Is your feature request related to a problem? Please describe.**
I do not believe this is a "problem", but a feature I would like.  For me and my players, syrinscape being played through the FoundryVTT just lags and is garbled.  However if we all install and use the Syrinscape online player, it works perfectly.  I still want to be able to control Syrinscape through Foundry, but not have the sound played through it.  I know you had a V4 beta which did this.  Controlled, but no sound in the browser.
**Describe the solution you'd like**
Please add a check box or something to the configuration to disable sound through the browser.

**Describe alternatives you've considered**
Muting Syrinscape in Foundry, but this does not really help as it still tries to play in both places and the online player gets signed out of.

**Additional context**
N/A

**Comment by frondeus** [comment]
It seems to be a duplicate of #50

**Comment by MythicalRollers** [comment]
You're correct, I did not even see that open.  Sorry to duplicate, but would really appreciate this feature.
